### ISSUE TITLE
Which speed solution is better?

### ISSUE THREAD

**Issue by n05ucc4u** [hingston/7-billion-humans-solutions#163]
the one that is a few ms faster (the score is the same), but a few commands longer or the one that is a few ms slower, but a few commands shorter?

**Comment by hingston** [comment]
I'd probably lean towards a few commands shorter just because it's a noticeable difference in the game. Anyone else have an opinion on this?

**Comment by n05ucc4u** [comment]
sure, however continuously improving solution by a few ms can eventually shave off the whole second (that's basically what happened with defrag disordered solution).

**Comment by NeunEinser** [comment]
The main problem I see with this is, how can you easily, and preferably automatically, tell the exact time in ms?
Also, I think all solutions would need to be reevaluated and show the exact time in the table.

**Comment by n05ucc4u** [comment]
there are a few commands that take 0.016s (for this game it's an indivisible time unit). the more of these you can insert at the start of the solution before the timer starts showing that it became slower - the faster your solution is.

**Comment by NeunEinser** [comment]
Ah, that's actually a clever way of measuring this.
I imagine, setting a mem value to some number would create that kind of delay?

**Comment by n05ucc4u** [comment]
yes, if myitem == myitem and telling to memory slot when it's empty also works.

**Comment by hingston** [comment]
Settled and written down in CONTRIBUTING.md: the size and speed the game reports are the only numbers used, so two
solutions showing the same time are treated as equally fast and the shorter one wins.

The delay trick described above is a genuinely clever way to measure the difference, but it is not something a
contributor can reasonably be asked to do for every submission, and the tables have no way to show a number the game
itself never displays. If someone wants to record sub-second timings properly, an extra column is possible — open a
new issue with a proposal.

Changes are in #313.
